### ISSUE TITLE
[Enhancement] Support for VoiceOver in Tab based navigation

### ISSUE THREAD

**Issue by sorenfruergaard** [kirbydesign/designsystem#999]
_NOT Prioritized_
For optimal support for VoiceOver this is needed.

_NOT UX Refined_
Doesn't need any UX.

**Is your enhancement request related to a problem? Please describe.**
When using VoiceOver (iOS) and navigating with tab bar, the navigation focus stays in the tab bar, after the user pushes a tab button, which makes  it hard for people using voiceover to navigate.

**Describe the solution you'd like**
The focus should be on the header or the first focusable input field.

**Describe alternatives you've considered**
none
